src/Commands/Env/ViewCommand.php

```diff
@@ -49,24 +53,8 @@ public function view($site_env, $options = ['print' => false,])
             return $url;
         }
 
-        // Otherwise attempt to launch it.
-        $cmd = '';
-        switch (php_uname('s')) {
-            case 'Linux':
-                $cmd = 'xdg-open';
-                break;
-            case 'Darwin':
-                $cmd = 'open';
-                break;
-            case 'Windows NT':
-                $cmd = 'start';
-                break;
-        }
-        if (!$cmd) {
-            throw new TerminusException("Terminus is unable to open a browser on this OS");
-        }
-        $command = sprintf('%s %s', $cmd, $url);
-        exec($command);
+        $this->getContainer()->get(LocalMachineHelper::class)->openUrl($url);
```

Comment:
I'd rather this helperwere accessible through
`$this->getHelper('local_machine')`
If you extend the application and override `getDefaultHelperSet`, you can add it.